### PR TITLE
Snackbar Notification

### PR DESCRIPTION
**Pull request by cskaandorp**
In this PR I have added the MUI Snackbar Notification that will be triggered if an email has been sent (create confirmed account / forgot password). The notification can be seen in the lower left corner and will be visible for 6 seconds before disappearing.

### CODE REVIEW

**Comment by cskaandorp** [comment]
Screenshot 2023-08-15 at 16 53 14
Screenshot 2023-08-15 at 16 53 42

Screenshots

**Review by J535D165 — APPROVED** [review]
Looks good! Did you style with prettier?

**Comment by cskaandorp** [comment]
I am pretty, and I am stylish. But I will make the code look better with prettier, haven't done it, sorry.

**Comment by cskaandorp** [comment]
Done. As far as I am concerned this PR is ready.

**Review by PeterLombaers — APPROVED** [review]
Looks good to me!

File: asreview/webapp/src/Components/SignUpForm.js [hunk 1]
```diff
@@ -109,7 +109,13 @@ const SignUpForm = (props) => {
 
   const { error, isError, mutate } = useMutation(BaseAPI.signup, {
     onSuccess: () => {
+      let email = formik.values.email;
       formik.setValues(initialValues, false);
+      if (typeof props.showNotification === "function") {
+        props.showNotification(
+          `An confirmation email has been sent to ${email}.`
```
Comment (Rensvandeschoot) [inline asreview/webapp/src/Components/SignUpForm.js:116]:
```suggestion
          `A confirmation email has been sent to ${email}.`
```

File: asreview/webapp/src/Components/SignUpForm.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ const SignupSchema = Yup.object().shape({
   password: Yup.string()
     .matches(
       /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*?&#])[A-Za-z\d@$!%*?&#]{8,}$/,
-      "Use 8 or more characters with a mix of letters, numbers & symbols",
+      "Use 8 or more characters with a mix of letters, numbers & symbols"
```
Comment (Rensvandeschoot) [inline asreview/webapp/src/Components/SignUpForm.js:76]:
```suggestion
      "Use 8 or more characters with a mix of letters, numbers, and symbols"
```

**Pull request merged**